yorkie/src/main/kotlin/dev/yorkie/document/crdt/RHTPQMap.kt

```diff
@@ -0,0 +1,161 @@
+package dev.yorkie.document.crdt
+
+import android.util.Log
+import dev.yorkie.document.time.TimeTicket
+import dev.yorkie.util.MaxPriorityQueue
+import dev.yorkie.util.PQNode
+
+
+/**
+ * RHTPQMap is replicated hash table with priority queue by creation time.
+ *
+ * @internal
+ */
+internal class RHTPQMap {
+    private val logTag = "RHTPQMap"
+
+    private val elementQueueMapByKey: MutableMap<String, MaxPriorityQueue<PQNode<TimeTicket, CrdtElement>>> =
+        mutableMapOf()
+    private val nodeMapByCreatedAt: MutableMap<String, RHTPQMapNode> = mutableMapOf()
+
+    /**
+     * `set` sets the value of the given key.
+     *
+     * @param key to get queue object from [elementQueueMapByKey]
+     * @param value The CrdtElement object to set [elementQueueMapByKey]
+     * @return CrdtElement if the object exist in [elementQueueMapByKey] by same key, it will return.
+     */
```

Comment:
How about following Kotlin's coding conventions for comments? 🤔

https://kotlinlang.org/docs/coding-conventions.html#documentation-comments
https://github.com/yorkie-team/yorkie-android-sdk/pull/7#discussion_r972180510

Reply:
I improved RHTPQMap's commentsby referring convention doc : 4492e34